sources/appendix-value-extraction.asciidoc

```diff
@@ -393,14 +400,14 @@ When detecting a regular element-level constraint (i.e. non type argument constr
 ** element of type `java.util.Collection`: (E) (as the type parameter `E` of `Collection` maps to `T` of `Iterable`, only the type parameter of the subtype is considered)
 ** `interface A<T> {}`, `interface B<U> {}`, `class C implements A<String>, B<Integer> {}`; element of type `C`: (T, U) (two non mapping type parameters)
 
-1. If `@ConstraintsApplyTo(WRAPPED_VALUE)` is given:
+1. If the constraint carries the `Unwrapping.Unwrap` payload:
 ** If no type parameter or more than one type parameter was found in step 2, raise a `ConstraintDeclarationException`
 ** Choose the most specific extractor matching the single type parameter found in step 2
 ** If there are several extractors which are equally specific, a `ConstraintDeclarationException` is raised.
 
-1. If `@ConstraintsApplyTo` is not given:
+1. If the constraint neither carries the `Unwrapping.Unwrap` nor the `Unwrapping.Skip` payload:
 ** If no type parameter or more than one type parameter was found in step 2, don't apply any value extractor
-** Choose the most specific extractor matching the single type parameter found in step 2 which is marked with `@ConstraintsApplyTo(WRAPPED_VALUE)`
+** Choose the most specific extractor matching the single type parameter found in step 2 which is marked with `UnwrapByDefault`
```

Comment:
Is it really what we want to do? I mean I would expect us to choose the most specific extractor AND unwrap only if it is marked with `UnwrapByDefault`.